changelog/unreleased/update-reva.md

```diff
@@ -0,0 +1,22 @@
+Enhancement: Update reva to v1.12
```

Comment:
Misses the `change` entry from https://github.com/cs3org/reva/pull/1982 if I'm not mistaken? 🤔

Reply:
I'm only listing the changes which are relevant for oCIS. I don't think copying all entries from the reva changelog to ocis makes much sense to be honest.

Reply:
Got it, thanks!